### PR TITLE
EDUCATOR-4753 - Adding an openedx.yaml file

### PR DESCRIPTION
**Pull request by jnlapierre**
Added an openedx.yaml to frontend-app-learner-portal-programs .

### CODE REVIEW

File: openedx.yaml [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# This file describes this Open edX repo, as described in OEP-2:
+# http://open-edx-proposals.readthedocs.io/en/latest/oeps/oep-0002.html#specification
```
Comment (nsprenkle) [inline openedx.yaml:2]: This link is out of date
```suggestion
# https://open-edx-proposals.readthedocs.io/en/latest/oep-0002-bp-repo-metadata.html#specification
```

File: openedx.yaml [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+nick: frontend-app-learner
+owner: mattcarter
+
+oeps:
+  oep-2: true
```
Comment (nsprenkle) [inline openedx.yaml:8]: Nit: capitalize to `True`

File: openedx.yaml [hunk 3]
```diff
@@ -0,0 +1,19 @@
+# This file describes this Open edX repo, as described in OEP-2:
+# http://open-edx-proposals.readthedocs.io/en/latest/oeps/oep-0002.html#specification
+
+nick: frontend-app-learner
+owner: mattcarter
```
Comment (nsprenkle) [inline openedx.yaml:5]: Updated spec says that this should absorb the team info.
```suggestion
owner:
  type: team
  value: masters-devs
```

File: openedx.yaml [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+  oep-7: true
+  oep-18: true
+  oep-30:
+    state: False
+    reason: TODO - Implement for this application if appropriate
+
+tags:
+  - portal
+
+supporting_teams:
```
Comment (nsprenkle) [inline openedx.yaml:18]: This key is deprecated and should be absorbed by the `owner` tag

**Comment by nsprenkle** [comment]
Just confirmed w/ a friend in the ARCH team that we should be using the most up-to-date formatting as provided here: https://open-edx-proposals.readthedocs.io/en/latest/oep-0002-bp-repo-metadata.html#specification

File: openedx.yaml (hunk 2 above)
Comment (mattcarter) [reply openedx.yaml]: this appears resolved now.  @nsprenkle  do you agree?

**Pull request merged**